adobecom/milo
# MWPW-132986 - Add spacing around library blocks

* Add spacing around blocks copied and pasted from the block library.

Blocks pasted to close to each other become difficult to edit and author. Updating the block library to add spacing above and below a block as it's pasted into the page should help authors avoid tables getting fused together in Word.


Resolves: [MWPW-132986](https://jira.corp.adobe.com/browse/MWPW-132986)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://block-spacing--milo--adobecom.hlx.page/?martech=off

Testing notes:

1. Open your Milo sidekick options. 
2. Update the branch your sidekick is pointing to, to point to `block-spacing` (this feature branch).
3. Save.
4. Go to a page in Sharepoint, open up the sidekick, launch the block library.
5. Copy and paste any block to the page. There should be new spacing above and below the block.


<img width="541" alt="image" src="https://github.com/adobecom/milo/assets/2539954/68c340b8-dd30-41b9-aca3-42e8912766a1">
